src/models/log.ts

```diff
@@ -3,8 +3,8 @@ import Joi from 'joi';
 const schema = Joi.object().keys({
   hashId: Joi.string().required().example('op09a'),
   userHashId: Joi.string().allow(null).required().example('b45zo0'),
-  objectType: Joi.string().allow(null).required().example('command'),
-  objectHashId: Joi.string().allow(null).required().example('ga9741s'),
+  objectType: Joi.string().allow(null).example('command'),
+  objectHashId: Joi.string().allow(null).example('ga9741s'),
```

Comment:
This model is only used for responses from the server. I think those will always contain these two properties, so they will still have to be required

Reply:
Updated